packages/types/src/interfaces/types.ts

```diff
@@ -1,11 +1,6 @@
 // Auto-generated via `yarn polkadot-types-from-defs`, do not edit
 /* eslint-disable */
 
-import {Api as ApiPromise, ApiRx} from "@cennznet/api";
-import {ApiTypes} from "@cennznet/api/types";
-import {AssetId} from "@cennznet/types/types";
-import {SubmittableExtrinsic as SubmittableExtrinsicBase} from "@polkadot/api/submittable/types";
-
```

Comment:
Moved the changes from here to index.js (one level up)
as every time we build/generate typedef, the types.ts will be overwritten